retailer-oas/retailer.yaml

```diff
@@ -679,6 +1090,62 @@ components:
             description: Indicates if the user can book this Time Slot.
             example: true
 
+    TimeSlotReservation:
```

Comment:
I am missing here an order ID, something to be able to ensure that the reservation of the timeslot has been made for a specific order (not sure it they will have an order ID at this time) you plan to attach it to the user id? how do you see the internal logic to check that a reservation belongs to an order?